utils/zoneinfo64/src/lib.rs

```diff
@@ -648,6 +648,19 @@ mod tests {
                 continue;
             }
 
+            // TODO: investigate why these zones don't work with jiff/tzdb-bundle-always
```

Comment:
hot tip: `cargo run -p tzif-inspect -- America/Ciudad_Juarez` from the temporal-rs repo will let you inspect tzif data.